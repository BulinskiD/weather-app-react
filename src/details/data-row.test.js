import React from 'react';
import DataRow from './data-row';
import {shallow} from "enzyme";

describe('DataRow', ()=>{

    const numericProps = {title: "test", data: 3};

    it('should render component correctly for given props', ()=>{
        const component = shallow(<DataRow {...numericProps} />)
        expect(component).toMatchSnapshot();
    });
});
