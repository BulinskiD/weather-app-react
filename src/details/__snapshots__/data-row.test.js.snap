// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataRow should render component correctly for given props 1`] = `
<Bootstrap(Row)
  className="mx-3 my-4"
>
  <Col
    as="div"
  >
    test
  </Col>
  <Col
    as="div"
  >
    3
  </Col>
</Bootstrap(Row)>
`;
